common-moonmage.lic

```diff
@@ -12,8 +12,24 @@ module DRCMM
     DRC.bput('predict weather', 'You predict that', 'You are far too', 'you lack the skill to grasp them fully', 'roundtime')
   end
 
-  def get_telescope
-    DRC.bput('get my telescope', 'you get', 'You are already holding that.', "That can't be picked up", 'You need a free hand to pick that up.', 'You should stop practicing your Athletics skill before you do that')
+  def get_telescope(storage)
```

Comment:
❌  Could you pull the `common-moonmage` changes into a separate PR? We want to merge it before the other PR to give people time to get the update.

Reply:
Yeah, will do that.